tests/AutoReview/FixerTest.php

```diff
@@ -262,6 +262,17 @@ public function provideFixerConfigurationDefinitionsCases()
         }, $fixers);
     }
 
+    /**
+     * @dataProvider provideFixerDefinitionsCases
+     */
+    public function testFixersReturnTypes(FixerInterface $fixer)
+    {
+        static::assertInternalType('int', $fixer->getPriority(), sprintf('Return type for ::getPriority of "%s" is invalid.', $fixer->getName()));
```

Comment:
shoulnd't this be checked by phpstan ?

Reply:
We probably need to increase analyse level before PHPStan spots this but yes, it will likely report that this assertion is superfluous at some point.